src/index.js

```diff
@@ -133,9 +133,14 @@ export default function commonjs ( options = {} ) {
 
 			resolveUsingOtherResolvers = first( resolvers );
 
-			entryModuleIdPromise = resolveId( options.input || options.entry ).then( resolved => {
-				entryModuleId = resolved;
-			});
+			const entryModules = [].concat( options.input || options.entry );
+			entryModuleIdsPromise = Promise.all(
+				entryModules.map( entry =>
+					resolveId( entry ).then( resolved => {
+						entryModuleIds.push( resolved );
+					})
+				)
+			);
```

Comment:
Minor totally unimportant nit: This looks a little more confusing as it needs to be (and it can possibly shuffle the order of the entry points which may or may not cause someone to stumble over this again in the future). If you `Promise.all` the mapped `entryModules` first and then assign the resulting array to `entryModuleIds`, the order would be preserved and one would not stop for a second to wonder if `entryModuleIds` really is a permuted map of `entryModules` 😉

Reply:
Totally fair,  I have updated the code to be more succinct and to use the result of the entryModuleIdsPromise.